packages/react-components/src/templates/Layout.tsx

```diff
@@ -69,6 +71,8 @@ const userButtonStyles = css({
     display: 'none',
   },
 
+  backgroundColor: paper.rgb,
```

Comment:
All areas on the left and top in the layout seem to be `paper` so this is not needed
(At least in the original designs like [this](https://projects.invisionapp.com/d/main#/console/20054351/425905082/preview), some other page designs since then have various inconcistencies

Reply:
This is the changes I had when the body was pearl, need to remove/adjust.